g2p/mappings/langs/font-encodings/fn_unicode.csv

```diff
@@ -1,21 +1,40 @@
+\uF000,x\u030c
 \uF101,c\u0313
-\uF102,č\u0313
+\uF102,c\u030c\u0313
```

Comment:
I used fully NFD forms here instead of the half-composed, half-decomposed value that was here before.

Reply:
This should make no functional difference in the end, since the mapping uses NFD normalization (which is applied to rules as well as inputs), but if you find it easier to read this way it's fine.

Reply:
Agreed. I have a mild preference for the half composed/half decomposed actually since the mapping takes care of normalization and it's a bit more readable, but I don't really care